my-uw/index.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: my-uw-template
+title: MyUW Style Guide
+slug: my-uw
+---
+<p class="lead">These are the styles for the redesign of MyUW. You can view the beta redesign by logging in with your NetID <a href="https://my.wisc.edu/portal/Login?profile=bucky" target="blank">here</a>. This page is a work in progress. These styles are all namespaced under the class 'my-uw', which affects everything outside the MyUW frame. The frame consists of the sidebar, header, and footer.</p>
```

Comment:
I think there is just confusion on what is meant by the frame. The _portal_ frame is not included in here. That is the top MyUW header with the search bar and crest, the sidebar and the footer. For now, that's not included since it is very customized. The _portlet_ frame is the outline of a portlet, and is included in the Toolkit. I'll nuance this paragraph's wording to hopefully make more sense.